data/boarding missions.txt

```diff
@@ -61,6 +61,160 @@ mission "Assisting Merchant (Large)"
 
 
 
+mission "Asteroid Course A"
+	assisting
+	minor
+	to offer
+		random < 10
+	source
+		government "Merchant" "Syndicate"
+		category "Light Freighter" "Interceptor"
+		not attributes "automaton"
+	destination "Earth"
+	on offer
+		conversation
+			`As you are approaching the <origin>, your radar sensors detect that the ship you are assisting is in the middle of a large asteroid's course! While in a normal flight this wouldn't cause much trouble, a ship without active shields and that has already suffered heavy damages could be completely destroyed.`
```

Comment:
I feel like this could be changed up a bit to increase the sense of urgency. Instead of saying that the ship is in the middle of the asteroid's course, say that the ship is about the be impacted/slammed by an asteroid, or that you spot a large asteroid coming right for the ship. Then for the explanation, I wouldn't say that it's because the ship is without shields and has already taken damage, because asteroids never impact the ship normally. Instead say something like since the ship is disabled it can't dodge.